src/com/sun/ts/tests/ejb30/timer/schedule/expression/annotated/ScheduleBean.java

```diff
@@ -217,10 +217,10 @@ public String attributeDefaults() {
   public String dayOfWeekOverDayOfMonth() {
     TimerInfo info = new TimerInfo("dayOfWeekOverDayOfMonth");
     Calendar cal = Calendar.getInstance();
-    cal.add(Calendar.DAY_OF_WEEK, 1);
+    cal.add(Calendar.DAY_OF_WEEK, 2);
     int nextDayOfWeek = TimerUtil.getForSchedule(Calendar.DAY_OF_WEEK, cal);
     // every year, every month, every hour, every minute, every second, but not
-    // every day. Only for the next day of week. So the timeout method for
+    // every day. Only for 2+ day of week. So the timeout method for
```

Comment:
What is wrong currently?

Why do we need to make this change?

Thanks,
Scott